File: Python/!PYTHON/PYTHONchallenges/Assessment/IT Computing/Year 10/a453 - programming/python files (.py)/calc.py
def calc():
    loop = False
    while loop == False:
        try:
            calc.menu = int(input("\nCalculater Program:\nChoose a option from the list.\n1) Add\n2) Subtract\n3) Times\n4) Divide\n>>> "))
            calc.numberOne = int(input("Enter your First number: "))
            calc.numberTwo = int(input("Enter your Second number: "))
            loop = True
        except:
            print("Only enter integers")
calc()
if calc.menu == 1: # Add
    answer = calc.numberOne + calc.numberTwo
    print(calc.numberOne,"+",calc.numberTwo,"=",answer)
if calc.menu == 2: # Subtract
    answer = calc.numberOne - calc.numberTwo
    print(calc.numberOne,"-",calc.numberTwo,"=",answer)
if calc.menu == 3: # Times
    answer = calc.numberOne * calc.numberTwo
    print(calc.numberOne,"*",calc.numberTwo,"=",answer)
if calc.menu == 4: # Divide
    answer = calc.numberOne / calc.numberTwo
    print(calc.numberOne,"/",calc.numberTwo,"=",answer)
    

File: C++/calc/calc.cpp
#include <iostream>

using namespace std;

int main() {
	int num1, num2, optr1;

	cout << "enter number 1:";
	cin >> num1;

	cout << "enter number 2:";
	cin >> num2;

	cout << "Menu:\n1. Add\n2. Subtract\n3. Times\n4. Divide\n>>> ";
	cin >> optr1;

	switch (optr1) {
	case 1:
		cout << num1 << " + " << num2 << " = " << num1+num2;

	case 2:
		cout << num1 << " - " << num2 << " = " << num1 - num2;

	case 3:
		cout << num1 << " x " << num2 << " = " << num1 * num2;

	case 4:
		cout << num1 << " / " << num2 << " = " << num1 / num2;

	}
}